tests/functional/filter/cram/filter-file-encoding-error.t

```diff
@@ -0,0 +1,33 @@
+Setup
+
+  $ source "$TESTDIR"/_setup.sh
+
+Create a metadata file that contains a non-ASCII character.
+
+  $ cat >metadata.tsv <<~~
+  > strain	col1
+  > SEQ_1	ã
+  > SEQ_2	b
+  > SEQ_3	c
+  > ~~
+
+Encode it as WINDOWS-1252.
+
+  $ iconv -f UTF-8 -t WINDOWS-1252 metadata.tsv > metadata-windows-1252.tsv
+
+The UTF-8 encoded file can be used without issues.
+
+  $ ${AUGUR} filter \
+  >  --metadata metadata.tsv \
+  >  --output-strains filtered_strains.txt
+  0 strains were dropped during filtering
+  3 strains passed all filters
+
+An error is shown when using the WINDOWS-1252 encoded file.
+
+  $ ${AUGUR} filter \
+  >  --metadata metadata-windows-1252.tsv \
+  >  --output-strains filtered_strains.txt
+  ERROR: File 'metadata-windows-1252.tsv' contains b'\xe3' which is not encoded as 'utf-8'.
+  Try saving the file using 'utf-8' encoding.
+  [2]
```

Comment:
This will be so so helpful for users compared to the standard error. :raised_hands: